bitnami/thanos/templates/alert-rule/replicate.yaml

```diff
@@ -4,13 +4,15 @@ SPDX-License-Identifier: APACHE-2.0
 */}}
 
 {{- /*
-Generated from https://github.com/thanos-io/thanos/blob/main/examples/alerts/alerts.md
+Generated from 'thanos-bucket-replicate' group from https://github.com/thanos-io/thanos.git
+Do not change in-place! In order to change this file run the script in
+https://github.com/bitnami/charts/tree/main/bitnami/thanos/hack
 */ -}}
 {{- if and .Values.metrics.enabled (or .Values.metrics.prometheusRule.default.create .Values.metrics.prometheusRule.default.replicate ) }}
 apiVersion: monitoring.coreos.com/v1
 kind: PrometheusRule
 metadata:
-  name: {{ template "common.names.fullname" . }}-replicate
+  name: thanos-bucket-replicate
```

Comment:
Name comes from the upstream - let me know if you think script should generated it as `{{ template "common.names.fullname" . }}-<suffix>` 
We would need to hardcode suffixes then, in order to accommodate  things like `-ruler`, `-store-gateway`.

Reply:
Hi,
  Sorry for the long delay.
  Yes, the name of the all kubernetes objects created by the chart should keep the naming. This would allow to have multiple instances with different names, and be coherent with how we handle the naming across all our charts.

Reply:
Kept `PrometheusRule` as it is. @rafariossaa let me know if there is anything else.